code/michael/html_css_flask/html_blog_lab_2/templates/blog.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" type="text/css" href="{{ url_for('static', filename='style.css') }}"/>
+</head>
+<body>
+    <header>
+        <div id="top">
+
+        <a href="blog.html"><img class="logo" src="../static/logo.png" alt="Logo"></a>
+        <p class="slogan"><em>Blogging is Life</em></p>
+        </div>
+        <div class="topnav">
+
+        </div>
+        <h1 class="title">BLOG</h1>
+
+    </header>
+    <div class="mid">
+        <div class="main">
+            <div class="content"><h2>Post 1</h2><h4>Today</h4> <div><img class="postimage" src="https://i2.wp.com/creapills.com/wp-content/uploads/2018/03/30-photos-absurdes-banques-dimages-23.jpg?resize=800%2C536&ssl=1" alt="stock image"><p>A clever caption</p></div><p class="contenttext">Lorem ipsum dolor sit amet consectetur adipisicing elit. Labore et ut eveniet recusandae, sint nam. Facilis libero mollitia esse saepe, earum, neque dolorum quos nihil consequuntur accusamus cupiditate est perspiciatis?</p></div>
+            <div class="content"><h2>Post 2</h2><h4>Yesterday</h4> <div><img class="postimage" src="https://www.pixsy.com/wp-content/uploads/2016/06/Cyber-woman-corn-closeup.jpg" alt="stock image"><p>A clever caption</p></div><p class="contenttext">Lorem ipsum dolor sit amet consectetur adipisicing elit. Labore et ut eveniet recusandae, sint nam. Facilis libero mollitia esse saepe, earum, neque dolorum quos nihil consequuntur accusamus cupiditate est perspiciatis?</p></div>
+            <div class="content"><h2>Post 3</h2><h4>Last Week</h4> <div><img class="postimage" src="https://edit.co.uk/uploads/2016/12/Image-2-Alternatives-to-stock-photography-Thinkstock.jpg" alt="stock image"><p>A clever caption</p></div><p class="contenttext">Lorem ipsum dolor sit amet consectetur adipisicing elit. Labore et ut eveniet recusandae, sint nam. Facilis libero mollitia esse saepe, earum, neque dolorum quos nihil consequuntur accusamus cupiditate est perspiciatis?</p></div>
+
+        </div>
+        <div class="sidenav">
```

Comment:
Missing semantic element <nav>, you can actually replace this <div> with <nav> and your code won't change but you'll still meet that requirement. <nav> defines that section as a navigation element.